### DISCUSSION TITLE
A couple of changes that might be of interest

### DISCUSSION THREAD

**Discussion by agspoon** [philosowaffle/ambientweather-local-server#17] [Ideas]
Thank you for sharing this tool.  It works like a charm with my AmbientWeather station (WS-2902A).  I've added a few fields to the Prometheus metrics, and fixed a couple of bugs.  I would like to share those with you, but don't know the best way to do so. I could just provide a git diff, if that works.

- Added Relative and Absolute barometric pressure
- Fixed up station UTC timestamp (sent as string, not number), and publish it as UNIX milliseconds from the Epoch, as that's what Grafana likes.
- Fixed spelling typo of "humiditiy_percent" to "humidity_percent"

**Comment by philosowaffle** [comment]
A Pull Request is welcome for these fixes/changes.  😃 

Perhaps you could also update the tables in [this document](https://github.com/philosowaffle/ambientweather-local-server/blob/main/docs/install/setup-ambientweather-console.md#1-console-firmware) to include your station model and what firmware you have installed?

**Comment by agspoon** [comment]
Looks good, thanks for taking the pull request, and the fix.
One thing I forgot to mention is that the example "grafana_dashboard.json" file has some fixed "uid:" values (BDhSj5Xnk), which caused errors for me when I originally imported it into grafana.

**Reply by levilliard** [reply]
Thanks for taking your time to make this tool.

I'm not familiar with Prometheus, I just run the AWLS on a docker container and i can confirm the server is running on my local nachine. I'm trying to use it with a WS-5000 station.

I'm not sure I undestand, what's the next step ? Install Prometheus ? Is there an URL to access a dashboard when it is running locally( something like 192.168.2.34:8001/dashboard) ?


Thanks in advance !

**Reply by philosowaffle** [reply]
Understanding OpenTelemetry and Prometheus is, to a degree, a pre-req to using this project.  To use a gross over simplification, imagine that all we're doing here is broadcasting some information on a new radio frequency. You still need to provide your own radios and speakers in order to listen to that radio channel.  In this simple example, something like [Grafana Agent](https://grafana.com/docs/agent/latest/) and [Mimir ](https://grafana.com/oss/mimir/) are your "radio", and something like [Grafana ](https://grafana.com/grafana/) is your "speaker".
